doc/using_tespy/connections.rst

```diff
@@ -18,28 +18,31 @@ As mentioned in the introduction, for each connection you can specify the follow
  * a balance closer for the fluid vector (fluid_balance).
 
 It is possible to specify values, starting values, references and data containers. The data containers for connections are dc_prop for fluid properties (mass flow, pressure, enthalpy, temperature and vapour mass fraction)
-and dc_flu for fluid composition. You need to import the :code:`hlp` module, if you want to specify data_containers.
+and dc_flu for fluid composition. If you want to specify data_containers, you need to import them from the :code:`tespy.tools` module.
 
 .. code-block:: python
 
+	from tespy.tools import dc_prop
+	from tespy.connections import ref
+
 	# set pressure and vapour mass fraction by value, temperature and enthalpy analogously
 	myconn.set_attr(p=7, x=0.5)
 
```

Comment:
I think it would be best to actually create a connection given two components, so if you copy the code into your editor, the python script will not raise errors.

Reply:
Wouldn't you need to create the components as well? These examples could become to complex for new users.

Reply:
Well, at no part have I written, that you have to create the components first... If we add that in the introduction to this part, it would be pretty clear, I think.